### PR TITLE
more tests with chirality and TR-braking

### PR DESCRIPTION
**Pull request by stepan-tsirkin**
* allowed the Chiral model to be left- or right-handed

* test that flipping handedness reverses berry_dipole

* test that TR flips AHC

### CODE REVIEW

File: tests/test_run.py [hunk 1]
```diff
@@ -197,3 +202,46 @@ def test_GaAs(check_run,system_GaAs_W90, compare_any_result,calculators_GaAs,Efe
...
+
+    grid_param={'NK':[10,10,4], 'NKFFT':[5,5,2]} 
+    check_run(system_Chiral_left , calculators_Chiral , fout_name="berry_Chiral" , suffix="left-run" ,
+                grid_param = grid_param,
+               parameters_K = {'_FF_antisym':True,'_CCab_antisym':True } , use_symmetry = True,
+            extra_precision = {"Morb":-1e-6},)
+
+
+def test_Chiral_leftTR(check_run,system_Chiral_left_TR, compare_any_result,Efermi_Chiral,compare_fermisurfer,calculators_Chiral):
+    "check that for time-revrsed model the ohmic conductivity is the same, but the AHC is opposite"
```
Comment (jaemolihm) [inline tests/test_run.py:226]: reversed

File: tests/test_run.py [hunk 2]
```diff
@@ -197,3 +202,46 @@ def test_GaAs(check_run,system_GaAs_W90, compare_any_result,calculators_GaAs,Efe
...
+    check_run(system_Chiral_left_TR ,calculators_Chiral , fout_name="berry_Chiral" , suffix="left-run" ,
+                grid_param = grid_param,
+               parameters_K = {'_FF_antisym':True,'_CCab_antisym':True } , use_symmetry = True,
+            extra_precision = {"Morb":-1e-6},
+            flip_sign = ['ahc']
+            )
+
+
+def test_Chiral_right(check_run,system_Chiral_right, compare_any_result,Efermi_Chiral,compare_fermisurfer,calculators_Chiral):
+    "check that for flipped chirality the ohmic conductivity is the same, but hte Berry dipole is opposite"
```
Comment (jaemolihm) [inline tests/test_run.py:237]: hte -> the

File: tests/test_run.py [hunk 3]
```diff
@@ -197,3 +202,46 @@ def test_GaAs(check_run,system_GaAs_W90, compare_any_result,calculators_GaAs,Efe
...
+    check_run(system_Chiral_left , calculators_Chiral , fout_name="berry_Chiral" , suffix="left-run" ,
+                grid_param = grid_param,
+               parameters_K = {'_FF_antisym':True,'_CCab_antisym':True } , use_symmetry = True,
+            extra_precision = {"Morb":-1e-6},)
+
+
+def test_Chiral_leftTR(check_run,system_Chiral_left_TR, compare_any_result,Efermi_Chiral,compare_fermisurfer,calculators_Chiral):
+    "check that for time-revrsed model the ohmic conductivity is the same, but the AHC is opposite"
+    grid_param={'NK':[10,10,4], 'NKFFT':[5,5,2]} 
+    check_run(system_Chiral_left_TR ,calculators_Chiral , fout_name="berry_Chiral" , suffix="left-run" ,
```
Comment (jaemolihm) [inline tests/test_run.py:228]: If these models do not take much time to run, what about just running left and left_TR calculation and comparing the results directly? Adding `flip_sign` option seems to make `check_run` adds complication to an already complicated function.

Comment (stepan-tsirkin) [reply tests/test_run.py:228]: Not sure, what you mean? t

1) to create another set of reference files for TR case?   

Or 

2) just run here both calculations, and compare them without reading reference files. i.e. comparing the result returned by `run()`, without accessing the files ?

Comment (jaemolihm) [reply tests/test_run.py:228]: I meant 2.

**Comment by jaemolihm** [comment]
@stepan-tsirkin if you are busy, I can finish this PR.

**Comment by stepan-tsirkin** [comment]
@jaemolihm , if you have time, please go ahead. Thanks.

**Review by jaemolihm — APPROVED** [review]
I made the changes. Please merge (I cant because the failing lint is required... maybe we should make this not required until we fix all formatting issues: can be done in settings only by the owner)

**Pull request merged**